pkg/hyperstart/types.go

```diff
@@ -186,6 +186,7 @@ type Container struct {
 	Process       *Process            `json:"process"`
 	RestartPolicy string              `json:"restartPolicy"`
 	Initialize    bool                `json:"initialize"`
+	BindMountDev  bool                `json:"bindMountDev"`
```

Comment:
This should be part of a more generic structure like:
```go
type IgnoredMounts struct {
        Dev bool
        Proc bool
        ...
}
```

Make this structure the way you want, but the point is that I don't want to see `Container` structure growing too much. And this is my feeling if we add a new field every time we have to handle a specific case.

Reply:
@sboeuf How about 
``` 
type IgnoredMountsInfo struct {
  BindMountDev bool    // Flag which says if -v /dev:/dev is true (just Dev does not make it clear at all to me)
  DevShmSize int   // For handing /dev/shm size in the future, currently hardcoded
}
```
I dont know what we will be doing about proc, or if we can do anything about it, so will leave that out for now, we can add those as we start handling them.

Reply:
Yes this looks good. I was only using `proc` as an example, but I was not expecting you to use it.
The case with `Shm` looks good and illustrates the fact that we can expect several special handling cases like this to come in the future.

Reply:
@sboeuf I have pushed changes. Have a look.